tests/cases/fourslash/docCommentTemplateClassDeclMethods01.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference path='fourslash.ts' />
 
 const enum Indentation {
-    Standard = 8,
+    Standard = 3,

```

Comment:
Why this change? It's unusual to set indentation to 3.

Reply:
Indentation is a really terrible name for it. It's actually the caret offset into the template. But indentation is the term consistently used for it. I could go in and change it all if warranted.

Reply:
Maybe in the old version this actually was an indentation?
This really makes no sense as an enum and could just be `const`s.